src/pages/index.tsx

```diff
@@ -216,136 +79,53 @@ const Home: NextPage<any> = () => {
   }, [user, geoJsonUsers]);
 
   useEffect(() => {
-    refetchMe();
-    refetchRecs();
-    refetchRequests();
-    refetchFavs();
     refetchGeoJsonUsers();
+    refetchMe();
```

Comment:
Same as on Sidebar.tsx. TRPC will automatically fetch when the component loads, and so there's no need to manually refetch in this useEffect. We can remove this useEffect, and remove the refetches in the TRPC calls above.

Reply:
ye

Reply:
Removing those results in recommendations, favorites, and users for the map not being fetched correctly when going when editing profile -> index. I'm going to keep them here